packages/utils/theme/tokens.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+export default {
+  heightPlaceCard: 152,
+  heightPlaceholder: 64,
+  heightSearchParamsSummary: 64,
```

Comment:
Do we need separate tokens for `heightSearchParamsSummary`, `heightTripSegment`, `heightTimelineInformation` if this value seems to be used only once and is closely connected just to a specific component?

Reply:
Or is your core idea just to collect all possible recurring values and sort everything later?

Reply:
Well for now we don't know if this value will be specific only for one component or it will be similar with something else. 

For example during this PR I find by accident that there are `PlaceCard` and `Placeholder` with the same `height`. I didn't know about it before but when I had two similar values in tokens, it was easy to find.

Btw Honza from Orbit wanted to collect in tokens every numbered value, but I think it's not so useful. We can start with something and we will see how it works in the future.

Reply:
Thanks for confirmation, that makes sense.
Just hoping that it will not bubble up that much, but we will see after a while ;) .